.github/workflows/format.yml

```diff
@@ -7,6 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
       - uses: subosito/flutter-action@v1
         with:
           flutter-version: "2.2.1"
```

Comment:
We concluded to always use the latest stable version. If you don't need any feature from `flutter analyze`, you can just use  [`dart analyze`](https://github.com/dart-lang/setup-dart).